docs/video-miners/how-to-guides/l2-migration.md

```diff
@@ -43,26 +43,22 @@ Other options are: [set up an Arbitrum node](https://developer.offchainlabs.com/
     
 2. **Follow the prompt to begin migration to Arbitrum**
     
-    Please note that if you are signing with the CLI and your connected wallet is NOT your orchestrator wallet, the stake amount will not appear until after you enter your Ethereum account address.
-    
-    <img src="/docs-assets/video-miners/how-to-guides/stake-info.png" alt="stake info" width="300" />
-
-
+    Please note that if you are signing with the CLI and your connected wallet is NOT your orchestrator wallet, the stake amounts will not be accurate until after you sign the transaction and we autodetect your orchestrator account from the signature.
+    <img src="/docs-assets/video-miners/how-to-guides/begin-migration.png" alt="begin migration" width="300"/>
```

Comment:
This prompt will only be visible if they're on Rinkeby. Should we also point out the link in the sidebar?

<img width="245" alt="Screen Shot 2022-01-24 at 11 55 25 PM" src="https://user-images.githubusercontent.com/555740/151034742-2938b528-0ce6-4568-92d2-075a0d4b632f.png">
?